### PR TITLE
Use ES operator 4.4

### PR DESCRIPTION
**Pull request by pavolloffay**
Signed-off-by: Pavol Loffay <ploffay@redhat.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -21,6 +21,7 @@ ES_OPERATOR_NAMESPACE ?= openshift-logging
 ES_OPERATOR_BRANCH ?= release-4.3
 PROMETHEUS_OPERATOR_TAG ?= v0.34.0
 ES_OPERATOR_IMAGE ?= quay.io/openshift/origin-elasticsearch-operator:4.3
+ES_PROXY_IMAGE ?= pavolloffay/elasticsearch-proxy:latest
```
Comment (pavolloffay) [inline Makefile]: TODO: change this once the image is published

**Comment by pavolloffay** [comment]
Build ES operator:
```
// cd eslasticsearch-operator, master branch
make the image
docker tag quay.io/openshift/origin-elasticsearch-operator pavolloffay/es-operator:feature-es6x 
docker push pavolloffay/es-operator:feature-es6x  
```

Build ES6 image, master branch https://github.com/openshift/origin-aggregated-logging/blob/master/elasticsearch/Dockerfile.centos7
```
docker build . -t pavolloffay/ecl-es6-latest -f Dockerfile.centos7
docker push  pavolloffay/ecl-es6-latest
```

Deploy ES operator
```
 ES_OPERATOR_IMAGE=pavolloffay/es-operator:feature-es6x ES_OPERATOR_BRANCH=master make deploy-es-operator 
```

Create Jaeger deployment with ES
```
cat <<EOF | oc apply -f -
# This CR deploys Jaeger and Elasticsearch
apiVersion: jaegertracing.io/v1
kind: Jaeger
metadata:
  name: simple-prod6
spec:
  strategy: production
  storage:
    type: elasticsearch
    elasticsearch:
      image: pavolloffay/ecl-es6-latest:latest
      nodeCount: 1
      storage: 
        storageClassName: gp2
        size: 5Gi
      resources:
        requests:
          cpu: 200m
          memory: 4Gi
        limits:
          memory: 4Gi
EOF
```

File: Makefile (hunk 1 above)
Comment (pavolloffay) [reply Makefile]: This does not seem to be needed at the moment as the ES operator has correct proxy (`latest`) and the tag for `4.4` does not exist https://quay.io/repository/openshift/origin-elasticsearch-proxy?tab=tags

```
    Environment:
      WATCH_NAMESPACE:      
      OPERATOR_NAME:        elasticsearch-operator
      PROXY_IMAGE:          quay.io/openshift/origin-oauth-proxy:v4.0.0
      ELASTICSEARCH_PROXY:  quay.io/openshift/origin-elasticsearch-proxy:latest
      ELASTICSEARCH_IMAGE:  quay.io/openshift/origin-logging-elasticsearch6:latest
```

Comment (pavolloffay) [reply Makefile]: I will change the value to `quay.io/openshift/origin-elasticsearch-proxy:latest` so we don't forget it when upgrading to 4.5

**Comment by pavolloffay** [comment]
To deploy the ES operator it's just enough to run this. ES operator image already has `4.4` tag but the release branch hasn't been updated to using master.
```
ES_OPERATOR_BRANCH=master make deploy-es-operator
```

**Comment by pavolloffay** [comment]
@kevinearls @objectiser could you please review?

**Comment by objectiser** [comment]
@kevinearls Did you want to test this?

**Comment by kevinearls** [comment]
@objectiser Tests are running now, I will update this when they finish.

**Pull request merged**